docs/usage/CATEGORY_SYNC.md

```diff
@@ -56,23 +56,27 @@ matched.
 final CategorySyncOptions categorySyncOptions = CategorySyncOptionsBuilder.of(sphereClient).build();
 ````
 
-The options can be used to provide additional optional configuration for the sync as well:
+Additional optional configuration for the sync can be configured on the `CategorySyncOptionsBuilder` instance, according to your need:
 - `errorCallBack`
-a callback that is called whenever an event occurs during the sync process that represents an error. Currently, these 
-events.
+a callback that is called whenever an event occurs during the sync process that represents an error.
 
```

Comment:
a callback that is called whenever an **Error** event occurs during the sync process